lib/gui/app/modules/drive-scanner.js

```diff
@@ -29,7 +29,9 @@ const permissions = require('../../../shared/permissions')
  * @type {String}
  * @constant
  */
-const BLOBS_DIRECTORY = path.join(__dirname, '..', '..', '..', 'blobs')
+const BLOBS_DIRECTORY = process.env.APPIMAGE && process.env.APPDIR
```

Comment:
Should we use the package.json's `.packageType` property here? i.e. `pkgJson.packageType === 'AppImage'`

Reply:
We use the `process.env.APPIMAGE` check in another place, is the `pkgJson.packageType` way better?

Reply:
I'd like to think it's more reliable due to us controlling it; the env tends to be the wild west on other people's computers ;)

Reply:
...and IIRC there is/was code that only copied selected env-vars into the environment of sub-processes?

Reply:
Here we go